readme.txt

```diff
@@ -217,7 +217,8 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 
 = [TBD] TBD =
 
-* Add - Added additional caching to TEC REST API archives and the Post Repository event/venue/organizer responses [117159]
+* Feature - Added additional caching to TEC REST API archives and the Post Repository event/venue/organizer responses [117159]
```

Comment:
Seems like this entry is from another PR?

Reply:
Correct, I've just changed _add_ to _feature_ per your feedback in yet another PR. 😎